src/ga_auth_handlers.cpp

```diff
@@ -619,6 +619,40 @@ namespace sdk {
         return state_type::done;
     }
 
+    //
+    // Sign PSBT
+    //
+    psbt_sign_call::psbt_sign_call(session& session, const nlohmann::json& details)
+        : auth_handler_impl(session, "sign_transaction")
```

Comment:
Should probably use `psbt_sign` instead of `sign_transaction` here, as auth handler actions are currently unique.

Reply:
Good catch, changed